torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -966,6 +953,39 @@ class PyLoweringContext {
           torch::lazy::Output(ir_value.node.get(), ir_value.index));
       lowering_ctx.AddResult(root);
     }
+
+    // add dummy parameter to cond xlacomputation's input for xla::while
+    // requriement
+    if (GetNameString() == "condctx") {
+      xla::XlaBuilder* local_builder = lowering_ctx.builder();
+      int64_t parameter_idx =
+          local_builder->GetProgramShape()->parameters_size();
+      int64_t additional_inputs_list_size = additional_inputs_list.size();
+      for (int64_t i = parameter_idx; i < additional_inputs_list_size; i++) {
+        XLATensorPtr xtensor = bridge::GetXlaTensor(additional_inputs_list[i]);
+        xla::Shape shape = xtensor->shape().get();
+        xla::XlaOp x = xla::Parameter(local_builder, parameter_idx, shape,
+                                      "UnusedArgumentsPlaceholder");
+        parameter_idx += 1;
+      }
+    }
+
+    // add dummy parameter to body xlacomputation's input for xla::while
+    // requriement
+    if (GetNameString() == "bodyctx" && additional_inputs_list.size() != 0) {
```

Comment:
couldn't you do
```
if (GetNameString() == "condctx") {
....
if (additional_inputs_list.size() != 0) {
...
}
}

```
?

Reply:
Thanks, combined and udpated to `if ((GetNameString() == "condctx") or (GetNameString() == "bodyctx" && additional_inputs_list.size() != 0)) {`

Reply:
Hmm, I don't see the change.

Reply:
thanks for review, rebase missed, updated now